gcc/rust/expand/rust-macro-builtins.cc

```diff
@@ -91,6 +91,9 @@ make_eager_builtin_invocation (
     case AST::BuiltinMacro::Include:
       path_str = "include";
       break;
+    case AST::BuiltinMacro::Option_env:
```

Comment:
You also need to add the variant to the `BuiltinMacro` enum in `rust-macro.h`. Also, this should be named `OptionEnv` instead of `Option_env` :)